inc/blocks/namespace.php

```diff
@@ -191,6 +220,25 @@ function register_post_type() {
 	);
 }
 
+/**
+ * Update the edit post link for XBs to be the parent post.
+ *
+ * @param string $link The original edit post link.
+ * @param int $post_id The post ID.
+ *
+ * @return string The updated edit post link.
+ */
+function update_xb_edit_post_link( string $link, int $post_id ) : string {
+	if ( get_post_type( $post_id ) !== POST_TYPE ) {
+		return $link;
+	}
+
+	$parent_id = wp_get_post_parent_id( $post_id );
+	$updated_link = $parent_id ? str_replace( $post_id, $parent_id, $link ) : $link;
```

Comment:
This should be ok but concerns me a bit for some reason. We could perhaps return early if there's no parent ID, and ideally we don't want to pass back the default edit link either, what might be a reasonable default? Empty string or null? If WP doesn't like that maybe link to the XB list view 🤔 

Besides that I'm thinking `add_query_arg()` might be a more intentional and concrete way to replace the existing post ID value, eg:
```suggestion
	if ( ! $parent_id ) {
		return '';
	}
	$updated_link = add_query_arg( [ 'post' => $parent_id ], $link );
```

Reply:
🤔
I guess an empty string makes sense as it would just keep you on the list view.